src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
lets use abstraction over implementation
```suggestion
        Machine truck = new Truck();
        Machine excavator = new Excavator();
        Machine bulldozer = new Bulldozer();
```